doc/releases/changelog-dev.md

```diff
@@ -180,6 +180,32 @@
   tensor([0.69301172, 0.67552491, 0.65128847], requires_grad=True)
   ```
 
+* Added new optimizer `qml.SPSAOptimizer` for noisy problems. It performs less device executions, due
+to the fact that it performs two measures to determine the gradient.
+[(#2661)](https://github.com/PennyLaneAI/pennylane/pull/2661)
+
+```python
+dev = qml.device("default.qubit", wires=1)
```

Comment:
Note that this code block needs to be indented with 2 space characters (see other examples in this file).